File: Array Snacks/question5.py
def numbers_on_even_positions(list):
	even_index_numbers = []
	for index in range(0, len(list)):
		if index % 2 == 0:
			even_index_numbers.append(list[index])
	return even_index_numbers

print(numbers_on_even_positions([1, 2, 3, 4, 5, 6, 7, 8, 9]))

File: Array Snacks/Question5.java
public class Question5{
   public int[] numbersOnEvenPositions(int[] list){
	
	int[] evenIndexNumbers = new int[(list.length + 1) / 2];

	int counter = 0;
	for(int index = 0; index < list.length; index++){
		if(index % 2 == 0){
			evenIndexNumbers[counter] = list[index];
			counter = counter + 1;
		}
	} 
	return evenIndexNumbers;
   }
}